src/app/data-explorer/querybuilder/querybuilder.component.html

```diff
@@ -0,0 +1,272 @@
+<!--
+Copyright 2022 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div class="query-builder">
+  <p>{{ title }}</p>
+  <textarea readonly class="output">{{ query | jql }}</textarea>
+  <div class="toggle-query-builder-div">
+    <button
+      class="toggle-query-builder-button"
+      type="button"
+      mat-button
+      color="primary"
+      (click)="toggleQueryBuilder()"
+    >
+      <mat-icon>{{ qbController.toggleQueryBuilderIcon }}</mat-icon>
+      {{ qbController.toggleQueryBuilderText }}
+    </button>
+  </div>
+  <!--<div *ngIf="qbController.showQueryBuilder">-->
+  <div class="box" [class.opened]="qbController.showQueryBuilder">
```

Comment:
The CSS solution was a quick fix to get the query builder to slide in and out of view. There is an angular approach to doing this, but it may require breaking the query builder down into different components. I didn't want to get bogged down making this work. I have created follow-up task gh-176 to address this.